pytorch_tabular/models/base_model.py

```diff
@@ -61,13 +68,16 @@ def _build_network(self):
 
     def _setup_loss(self):
         if self.custom_loss is None:
-            try:
-                self.loss = getattr(nn, self.hparams.loss)()
-            except AttributeError as e:
-                logger.error(
-                    f"{self.hparams.loss} is not a valid loss defined in the torch.nn module"
-                )
-                raise e
+            if self.hparams.ssl_task == "Contrastive":
+                self.loss = loss_contrastive
```

Comment:
@Actis92  Do you think its a good idea to also issue a warning to the user that whatever loss they might have set will be ignored in an SSL task? Or may be include it in the docs that loss will be ignored in SSL. what do you think?